big_tests/tests/inbox.hrl

```diff
@@ -0,0 +1,8 @@
+-record(conv, {
```

Comment:
Why record? I have this impression this was a map before rebasing, am I right?

Reply:
No, it has always been a record. :)